lib/ansible/modules/network/aci/aci_bd_to_l3out.py

```diff
@@ -18,7 +18,7 @@
 - Bind Bridge Domain to L3 Out on Cisco ACI fabrics.
 notes:
 - The C(bd) and C(l3out) parameters should exist before using this module.
-  The M(aci_bd) and M(aci_l3out) can be used for these.
+  The M(aci_bd) and M(aci_l3out_route_tag_policy) can be used for these.
```

Comment:
That is incorrect. It really should be **aci_l3out**, which does not exist as a module yet, but is in the pipeline.

Currently users build the l3out using **aci_rest**. We have a ticket open wrt. this, the documentation is based on the legacy modules which did include the **aci_l3out** module but was pretty limited in functionality. We are waiting for ACI experts to guide us in how they see this work in the future.